doc/method/results.md

```diff
@@ -113,11 +94,11 @@ each HHDB hemihomonym, in the right places]
 
 ### Ongoing update
 
-Building OTT version 2.11 from sources requires 11 minutes 42 second of real time [KC: and memory?].
+Building OTT version 2.11 from sources requires 11 minutes 42 second of real time. Our process currently runs on a machine with 16GB of memory, and 8GB is not sufficient.
 
-In the upgrade from 2.10 to 2.11, we added new versions of both NCBI and GBIF. NCBI updates frequently, so changes tend to be minimal and incorporating the new version was trivial. In contrast, the version from GBIF represented a major change in their taxonomy synthesis method. The file format changed, requiring changes in our import code. In addition, many taxa disappeared, requiring changes to our ad hoc patches during the normalization stage.  
+In the upgrade from 2.10 to 2.11, we added new versions of both NCBI and GBIF. NCBI updates frequently, so changes tend to be minimal and incorporating the new version was trivial. In contrast, the version from GBIF represented both a major change in their taxonomy synthesis method. Many taxa disappeared, requiring changes to our ad hoc patches during the normalization stage. In addition, the new version of GBIF used a different taxonomy file format, which requires extensive changes to our import code (most notably, handling taxon name-strings that now included authority information).
 
-* [KC: estimate on number of curation hours required for 2.10 -> 2.11 update?]
+We estimate the the update from OTT v2.10 to OTT v3.0 required approximately 3 days of development time. This was much greater than previous updates due to the changes required to handle the major changes in GBIF content and format.  
```

Comment:
hmm.  the 3 days was for the GBIF and NCBI updates.  but a lot of other work was involved in the 'update' that was not directly related to the source taxonomy update, e.g. the changes to alignment.  will reword later